source/languageHandler.py

```diff
@@ -23,6 +24,12 @@
 LOCALE_SLANGDISPLAYNAME=0x6f
 LOCALE_USER_DEFAULT = 0x400
 LOCALE_CUSTOM_UNSPECIFIED = 0x1000
+LOCALE_SENGLISHLANGUAGENAME = 0x00001001
+LOCALE_SENGLISHCOUNTRYNAME = 0x00001002
+LOCALE_IDEFAULTANSICODEPAGE = 0x00001004

```

Comment:
Would you be able to turn these new constants into an `IntEnum`?

Considering the amount of work that has gone into this already, and the complexity of this PR, I'm happy to create an issue to move the other related constants into the `IntEnum`, and prepare the deprecations myself.

Reply:
Done as of d2b973636. The PR description has been updated with an appropriate change log entry for developers. I haven't moved The remaining module level constants such as `LOCALE_CUSTOM_UNSPECIFIED` to the enum since they're not used when retrieving locale info they just  happens to share the same prefix `LOCALE`